docs/Releasing.md

```diff
@@ -4,16 +4,38 @@ Open Enclave SDK Release Procedures
 This document covers how we handle creating a release. Let's look at it from an
 example:
 
+Selecting a Release Manager
+---------------------------
+
+A committer can be nominated to become the release manager by drafting a pull
+request that sets the "Release" tag to the "Area" of one of the committers in
+[Committers.md](Committers.md). The submitter of that nomination should then
+send a link to that pull request through email to all of the committers for open
+discussion. This discussion & PR should be open for at least 3 days, and people
+can make their votes known in the pull request itself by either approving the
+pull request or requesting changes. If there is any opposition from the
```

Comment:
I think there's one bug in the process proposed, which is that it doesn't talk about what happens if there's two proposals.   For example, what happens if there's two different PR's with different candidates, and no one opposes either one, just each has different people voting for them.   I think that's another case where the CGC should take up the issue.

And if there are multiple competing candidates (e.g., candidates from different companies), should the identity of who votes for whom be public or private?   For decisions that affect personnel, there are good arguments for keeping votes private.

I have no objection to the current text as interim process, but I think all of the above considerations need to be taken into account for the non-interim process.

Reply:
Yes, I agree. In the future, we may wish to call for an election among the committers with a deadline to get your votes in, or something along those lines.

Reply:
I have a lot of questions as to whether this needs to be documented as a special process:
- Why is the process of selecting a Release Manager not just the standard process of the CGC making that an agenda item with the usual standard criteria for voting? 
    - I see the Release Manager as an operational role that the CGC should decide, not something that all committers elect.
    - As in the case of suggesting new committers, all committers are free to suggest a change in the Release Manager, and it becomes a standard order of business for the CGC to take up and decide on.
        - Is the codification of discussion duration etc. more broadly applicable to how the CGC makes decisions? Or is electing a Release Manager intended to be a higher bar for some reason?

Specifically, the heightened process seems to be an imply that the Release Manager serves some kind of elevated role, which is probably something we do not want to endow to an individual. I expect that a Release Manager is not a gatekeeper, just a coordinator for a release, with triage authority ultimately rooting to a technical steering committee of some kind.

I think it's detrimental to invest in a heavyweight process of electing a Release Manager such that it makes backups or handing off the role difficult. e.g. Release Manager gets run over by a bus the day on a day we announce a CVE and a patch update. I want the role to be sufficiently transferrable that someone can quickly step up, read the doc and get the release back on track.

Reply:
@dthaler @CodeMonkeyLeet I'm good with the idea that the CGC decides on the release manager, instead of calling for a vote of the committers. It really does simplify process, when it's not necessary to complicate it. I will update this document accordingly